src/main/java/core/basesyntax/FileCreator.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class FileCreator {
+
+    public static void createFile() {
+        String fileName = "/Users/Test_user/IdeaProjects/jv-fruit-shop/reportToRead.csv";
```

Comment:
Avoid using the absolute path to a resource. It is recommended to use relative paths so that everyone who pulls your project can run it without modification.